drivers/video/ov7725.c

```diff
@@ -560,32 +560,23 @@ static const struct video_driver_api ov7725_driver_api = {
 
 static int ov7725_init(const struct device *dev)
 {
-	struct ov7725_data *drv_data = dev->data;
 	const struct ov7725_config *cfg = dev->config;
 	struct video_format fmt;
 	uint8_t pid, ver;
 	int ret;
 
-
-	if (drv_data->reset_gpio) {
-		ret = gpio_pin_configure(drv_data->reset_gpio,
-					 drv_data->reset_pin,
-					 GPIO_OUTPUT_ACTIVE |
-					 drv_data->reset_flags);
-		if (ret) {
-			return ret;
-		}
-
-		gpio_pin_set(drv_data->reset_gpio,
-			     drv_data->reset_pin,
-			     0);
-		k_sleep(K_MSEC(1));
-		gpio_pin_set(drv_data->reset_gpio,
-			     drv_data->reset_pin,
-			     1);
-		k_sleep(K_MSEC(1));
+#if DT_INST_NODE_HAS_PROP(0, reset_gpios)
```

Comment:
not multi-instance friendly

Reply:
As @bbilas said, the driver isn't multi-instance friendly to begin with.  I'm also guessing its unlikely one would have more than one of these devices in a system.  (And if/when someone has such a system they will hopefully update the driver to support that).